### ISSUE TITLE
Can't approve User

### ISSUE THREAD

**Issue by spiritblue01** [dani-garcia/vaultwarden#4018]
### Subject of the issue
I invited a user and the user registered. Afterwards I can't approve it there is coming an error message:

"null is not an object (evaluating 't.buffer')"

### Deployment environment

* vaultwarden version: (Web-Tresor) Version 2023.7.1

* Install method: Docker

* Clients used: Safari, Chrome

* Reverse proxy and version: no

**Comment by BlackDex** [comment]
Please try the `testing` tagged image and see if that does work.

**Comment by spiritblue01** [comment]
Thank for the answer, but where can I find this image?

**Comment by BlackDex** [comment]
You can use latest now

**Comment by BlackDex** [comment]
@spiritblue01 any update on this with the current new 1.30.x version?
Also, we would need some logs and screenshots, else we can't help or debug anything.

I tested these steps just a few minutes ago during development, and they seem to work just fine.
But, without logs of your situation, we can't determine anything.

**Comment by BlackDex** [comment]
Closing this as stale. If needed please comment and re-open

**Comment by backspace** [comment]
This is happening for me too, I was on 1.27.0 but have updated to 1.31.0 to no avail.

When I want to confirm the person the fingerprint phrase is missing:

<img width="520" alt="Members | Vaultwarden Web Vault 2024-07-09 16-42-53" src="https://github.com/dani-garcia/vaultwarden/assets/43280/66ccc03f-2a0a-429d-900d-ea449ad0f051">

(But they do have one, they tell me.)

This is what shows if I try to confirm anyway:

<img width="320" alt="Members | Vaultwarden Web Vault 2024-07-08 11-45-26" src="https://github.com/dani-garcia/vaultwarden/assets/43280/bfbdbd09-26fa-4d11-83a4-9fe12d2dafc4">

Maybe of interest is that I had to delete their account because they forgot their password.

When I tried it with a different account that I had not deleted, the fingerprint phrase did show.

I’ve installed the testing Docker image, what do I need to collect to help debug? Should I delete the account again?
